ENIGMAsystem/SHELL/Graphics_Systems/None/NONEStd.cpp

```diff
@@ -0,0 +1,95 @@
+/** Copyright (C) 2017 Faissal I. Bensefia
+*** Copyright (C) 2008-2013 Robert B. Colton, Adriano Tumminelli
+*** Copyright (C) 2014 Seth N. Hetu
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <stdlib.h>     /* malloc, free, rand */
+#include <unordered_map>
+#include <math.h>
+
+using namespace std;
+#include "OPENGLStd.h"
+#include "Universal_System/shaderstruct.h"
```

Comment:
My guess is that this entire file can be deleted, but if not, the vast majority of these includes can.